src/util/Cache.h

```diff
@@ -274,7 +263,8 @@ class FlexibleCache {
     // adapt the sizes of the pinned and non-pinned part of the cache
     auto sz = _valueSizeGetter(*valuePtr);
     _totalSizeNonPinned -= sz;
-    _totalSizePinned += sz;
+    _totalSizePinned =
+        MemorySize::bytes(_totalSizePinned.getBytes() + sz.getBytes());
```

Comment:
Shouldn't be necessary anymore with all your nice operators.

Reply:
Done, see [aa91fc6](https://github.com/ad-freiburg/qlever/pull/1111/commits/aa91fc6876a02bdece67b4c9b687c8abf2a92296).